docs/site/content/en/docs/Tutorials/grafana/index.md

```diff
@@ -85,31 +85,21 @@ The `/labelValues` endpoint supports a `filter` query parameter to filter out da
 There are 2 ways to filter:
 1. provide a json object that must exist in the label values.
    
-   For example, if `version` is a label we can pass in `{"version":"1.2.3"}` to only include datasets where the version value is `1.2.3` 
+   For example, if `version` and `txRate` are label names then `{"version":"1.2.3"}` will only include labelValues where `version=1.2.3` 
+   and `{"versoin":"1.2.3","txRate":2000}` will add the `txRate=2000` requirement.
```

Comment:
Who should the the "resolver" the person who asked or the person who answered? I think "asked" as it can be offending when someone replies but does not properly address the query.